security/src/main/java/de/evoila/cf/security/keystore/KeyStoreHandler.java

```diff
@@ -0,0 +1,117 @@
+package de.evoila.cf.security.keystore;
+
+import org.bouncycastle.util.io.pem.PemObject;
+import org.bouncycastle.util.io.pem.PemReader;
+import org.springframework.stereotype.Service;
+import sun.security.util.DerInputStream;
+import sun.security.util.DerValue;
+
+import javax.naming.ConfigurationException;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.StringReader;
+import java.math.BigInteger;
+import java.security.GeneralSecurityException;
+import java.security.KeyFactory;
+import java.security.KeyStore;
+import java.security.PrivateKey;
+import java.security.cert.Certificate;
+import java.security.cert.CertificateFactory;
+import java.security.spec.PKCS8EncodedKeySpec;
+import java.security.spec.RSAPrivateCrtKeySpec;
+import java.util.Base64;
+
+/**
+ * Created by reneschollmeyer, evoila on 21.11.18.
+ */
+@Service
```

Comment:
I am not sure if it makes sense, that this is a Service. The Keystore is managed centrally for the application/platform below anyway if I remember correctly. I would make more sense to have a Singleton for it, what you think?

Reply:
I agree